modules/project_cleanup/function_source/main.go

```diff
@@ -338,6 +403,48 @@ func invoke(ctx context.Context) {
 		return tagKeyCreatedAt.Before(resourceCreationCutoff)
 	}
 
+	projectDeleteRequestedFilter := func(projectID string) bool {
+		p, err := cloudResourceManagerService.Projects.Get(projectID).Context(ctx).Do()
+		if err != nil {
+			logger.Printf("Failed to get project [%s], error [%s]", projectID, err.Error())
+			return false
+		}
+		if p.LifecycleState == "DELETE_REQUESTED" {
+			return true
+		}
+		return false
+	}
+
+	removeSCCNotifications := func(organization string) {
+		logger.Printf("Try to remove SCC Notifications from organization [%s]", organization)
+		req := &securitycenterpb.ListNotificationConfigsRequest{
+			Parent: fmt.Sprintf("organizations/%s", organization),
```

Comment:
It looks like default it returns 10 but configurable to 1000? Could we request a higher number?

Reply:
https://pkg.go.dev/cloud.google.com/go/securitycenter/apiv1/securitycenterpb#ListNotificationConfigsRequest

Reply:
the quota is [500 per organization](https://cloud.google.com/security-command-center/quotas#notifications)
I will use this value for the page size

Reply:
fixed